build/mbl/build.sh

```diff
@@ -918,6 +925,15 @@ while true; do
     ;;
 
   inject)
+    if [ -n "${inject_key_files:-}" ]; then
+      for machine in $machines; do
+        for file in $inject_key_files; do
+          base="$(basename "$file")"
+          cp "$file" "$builddir/machine-$machine/mbl-manifest/build-$distro/$base"
```

Comment:
I think we should ensure that the copied keys will have the expected filenames set by `MBL_ROT_KEY_FILENAME` and `MBL_FIT_ROT_KEY_FILENAME` variables set in `meta-mbl/meta-mbl-bsp-common/conf/include/mbl-default.conf`. And looking at this conf file, we should change `MBL_ROT_KEY_FILENAME ?= "rot_key.pem"` to `MBL_ROT_KEY_FILENAME = "rot_key.pem"`.